drizzle-orm/src/mysql-core/dialect.ts

```diff
@@ -323,13 +323,15 @@ export class MySqlDialect {
 
 		const valuesSql = sql.fromList(valuesSqlList);
 
+		const ignoreSql = ignore ? sql`insert ignore` : sql`insert`
+
 		const returningSql = returning
 			? sql` returning ${this.buildSelection(returning, { isSingleTable: true })}`
 			: undefined;
 
 		const onConflictSql = onConflict ? sql` on duplicate key ${onConflict}` : undefined;
 
-		return sql`insert into ${table} ${insertOrder} values ${valuesSql}${onConflictSql}`;
+		return sql`${ignoreSql} into ${table} ${insertOrder} values ${valuesSql}${onConflictSql}`;
```

Comment:
It's better to do ``sql`insert${ignoreSql}...` `` and use ``sql` ignore` `` or `sql.empty()`/`undefined` for `ignoreSql`. It's more verbose that way.